draft-ietf-ohai-ohttp.md

```diff
@@ -1044,8 +1044,9 @@ Encapsulated Response.
 A relay MAY add information to requests if the client is aware of the nature of
 the information that could be added.  The client does not need to be aware of
 the exact value added for each request, but needs to know the range of possible
-values the relay might use.  It is important to note that information added by
-the relay can reduce the size of the anonymity set of clients at a gateway.
+values the relay might use.  Importantly, information added by the relay - beyond
+what is already revealed through encapsulated requests from clients - can reduce
```

Comment:
```suggestion
what is already revealed in the content of a request - can reduce
```

Reply:
I think we ought to keep this as is, since the motivating reason for this change was that the config used to encapsulate a request can partition the client anonymity set.